packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -92,66 +94,69 @@ Cons:
 
 ### API
 
-#### Icon column element:
-
-_Component Name_
-
--   `nimble-table-column-icon`
-
-_Props/Attrs_
-
--   `field-name`: string
--   `key-type`: 'string' | 'number' | 'boolean'
-
-_Content_
-
--   column title (icon)
--   1 or more `nimble-mapping-icon` or `nimble-mapping-spinner` elements
-
-#### General mapping column element:
+#### Enum column element:
 
 _Component Name_
 
--   `nimble-table-column-enum-text`
+-   `nimble-table-column-enum`
 
 _Props/Attrs_
 
 -   `field-name`: string
 -   `key-type`: 'string' | 'number' | 'boolean'
--   `fractional-width`: number (defaults to 1)
--   `min-pixel-width`: number (defaults to minimum supported by table)
+-   `fixed-to-icon-width`: boolean - When set, the column will have a fixed width that makes the column the appropriate width to render only a single icon in the cell. This should only be set when the header contains a single icon (no text) and none of the child mapping elements will result in text being rendered in a cell.
```

Comment:
If the specificity in the attribute name is there to enable API expansion, i.e. `fixed-to-something-else-width`, then maybe we make it an enum off the bat?

```ts
TableColumnEnumFixedWidthMode {flexibleWidth: undefined, iconWidth: 'icon-width'}
```

```html
fixed-width-mode="icon-width"
```